lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -1770,18 +1764,6 @@ def test_divided_within_group(self, check_condition_function):
             'can_vote': True,
             'can_report': True
         })
-        RequestCache.clear_all_namespaces()
```

Comment:
The following test stopped passing once I fixed the course modes above. I spoke with AJ, the original author, who had no memory of what this is for. Given that it has been broken since it was written I'm just removing it.

Reply:
What are the permissions that come back?

Reply:
IIRC it's the same as the ones further up above:

'editable': False,
'can_reply': True,
'can_delete': False,
'can_openclose': False,
'can_vote': True,
'can_report': True